legend-engine-pure-code-compiled-core/src/main/resources/core/pure/test/testable.pure

```diff
@@ -31,7 +31,7 @@ Class <<typemodifiers.abstract>> meta::pure::test::AtomicTest extends Test
 Class <<typemodifiers.abstract>> meta::pure::test::TestBatch
 {
     id         : String[1];
-    batchId    : Integer[0..1];
+    batchId    : Integer[1];
```

Comment:
given this property will be required, should we change the protocol class to reflect this, and change from object Integer to primitive int?

https://github.com/finos/legend-engine/blob/master/legend-engine-protocol-pure/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/test/TestBatch.java#L25

Reply:
Definitely makes sense. Will update.